EventFilter/EcalRawToDigi/plugins/UnpackGPU.cu

```diff
@@ -0,0 +1,340 @@
+#include "EventFilter/EcalRawToDigi/interface/ElectronicsIdGPU.h"
+
+#include "UnpackGPU.h"
+
+namespace ecal {
+  namespace raw {
+
+    __forceinline__ __device__ void print_raw_buffer(uint8_t const* const buffer,
```

Comment:
are these debugging functions? If so, I imagine putting them in a central place can be helpful.

Reply:
@jpata what do you mean by a central place? The functions are probably specific to the unpacker even if they do something rather generic. Not sure if they are useful elsewhere.

Reply:
I don't think there would be other users at the moment.